src/test/json/scim-patch/remove_multiple_by_value.json

```diff
@@ -0,0 +1,43 @@
+{
+  "schemas": [
+    "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+  ],
+  "Operations": [
```

Comment:
Tests for combination of multi-value remove operation and multiple operations in the same request.